student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom, age, name, parent_permission)
```

Comment:
- [ ] the `name` and `parent_permission` arguments should have a default value similar to in the `Person` class